ext/vulnsrc/rhel/rhel.go

```diff
@@ -184,18 +185,30 @@ func parseRHSA(ovalReader io.Reader) (vulnerabilities []database.VulnerabilityWi
 	for _, definition := range ov.Definitions {
 		pkgs := toFeatures(definition.Criteria)
 		if len(pkgs) > 0 {
+
+			// Init vulnerability
 			vulnerability := database.VulnerabilityWithAffected{
 				Vulnerability: database.Vulnerability{
-					Name:        name(definition),
-					Link:        link(definition),
 					Severity:    severity(definition),
 					Description: description(definition),
 				},
 			}
 			for _, p := range pkgs {
 				vulnerability.Affected = append(vulnerability.Affected, p)
 			}
-			vulnerabilities = append(vulnerabilities, vulnerability)
+
+			// Only RHSA is present
+			if len(definition.References) == 1 {
```

Comment:
If we default to rhsaName and rhsaLink, we could make this code a little more simple by making it along the lines of

```go
// Prefer the CVE name and link if available.
if len(definition.References) > 1 {
  for _, ref := range definition.References {
    if ref.Source == "CVE" {
      vulnerability.Name = ref.ID
      vulnerability.Link = ref.URI
    }
  }
}
```

Reply:
Let's take the same example as above:

```
>   <reference source="RHSA" ref_id="RHSA-2015:1207-00" ref_url="https://rhn.redhat.com/errata/RHSA-2015-1207.html"/>
>   <reference source="CVE" ref_id="CVE-2015-2722" ref_url="https://access.redhat.com/security/cve/CVE-2015-2722"/>
>   <reference source="CVE" ref_id="CVE-2015-2724" ref_url="https://access.redhat.com/security/cve/CVE-2015-2724"/>
>   <reference source="CVE" ref_id="CVE-2015-2725" ref_url="https://access.redhat.com/security/cve/CVE-2015-2725"/>
```

If we do that we will only create one vulnerability (CVE-2015-2725) and lose the data for the 2 others CVE.